src/App.vue

```diff
@@ -1,22 +1,32 @@
 <template>
-  <GitHub v-if="github" />
+  <AllOpenIssues v-if="github" />
```

Comment:
you can not change App.vue

Reply:
Should it route to new page if username and password are correct? @MKRNaqeebi

Reply:
no, route use components
just hide the login component and render the new component